### DISCUSSION TITLE
Server Error opening devices after upgrade netbox 3.5.4 to 3.5.5

### DISCUSSION THREAD

**Discussion by echicor** [netbox-community/netbox#13244] [Help Wanted!]
After updating from netbox 3.5.4 to 3.5.5, I can no longer open a device via the interface or even edit or create one. Unfortunately, the error message "Server Error" always appears with the following explanation: 

> <class 'AttributeError'>
> 
> 'ImageDraw' object has no attribute 'textsize'.
> 
> Python version: 3.8.10
> NetBox version: 3.5.5

can someone help me to solve this problem? I can provide more details about the installation if needed

Translated with www.DeepL.com/Translator (free version)

**Comment by abhi1693** [comment]
See #13115
